dts/bindings/fuel-gauge/battery-cutoff.yaml

```
@@ -0,0 +1,29 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+
+  Properties for fuel-gauges that may control battery cutoff, this is common in SBS-compliant or
+  similarly smart battery fuel gauges.
+
+  Note: These properties are to be used with meaningful defaults in fuel gauge ICs that can cut off
+  their associated battery from the system. See the default fuel gauge SBS Gauge compatible as an
+  example.
+
+properties:
+  battery-cutoff-support:
+    description: |
+      Helper prop that indicates whether this device can cutoff the battery; this is also often
+      referred to as ship or sleep mode. Any non-zero value means battery cutoff is supported.
```

This is now a boolean property, so the "non-zero" comment no longer applies.
```suggestion
      referred to as ship or sleep mode.
```